app/views/admin/search/_purchases.html.erb

```diff
@@ -40,5 +40,15 @@
     </tbody>
   </table>
 
-  <%= paginate @purchases, views_prefix: "admin" %>
+  <% if @pagination&.pages && @pagination.pages > 1 %>
```

Comment:
could we use any of the pagy view helpers here? if not could we extract this custom bit into a partial?